<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QRB56CU" name="Default test" tests="1" failures="1" timestamp="13 Feb 2024 18:59:55 GMT" time="4.679" errors="0">
  <testcase name="runScenario" time="4.428" classname="runners.TestRunner_TestNG">
    <failure type="java.lang.AssertionError" message="expected [201] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [400]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at stepDefinitions.Steps_add_remove_books.bookIsAdded(Steps_add_remove_books.java:81)
	at âœ½.The book is added(file:///C:/Users/jawad/eclipse-workspace/RestAssured_Cucumber/src/test/resources/featureFiles/end2endTest.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
